ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1285,13 +1282,14 @@ Rectangle {
                             onTextChanged: {
                                 if (length <= control.messageLimit) {
                                     var symbols = ":='xX><0O;*dB8-D#%\\";
-                                    if ((length > 1) && (symbols.indexOf(getText((cursorPosition - 2), (cursorPosition - 1))) !== -1)
-                                            && (!getText((cursorPosition - 7), cursorPosition).includes("http"))) {
+                                    var symbolIdx = ((cursorPosition > 2) && (symbols.indexOf(getText((cursorPosition - 2), (cursorPosition - 1)))!==-1))
+                                                    ? (cursorPosition -1) : (symbols.indexOf(getText(0, 1))!==-1) ? 2 : -1;
+                                    if ((getText(symbolIdx-2, (symbolIdx-1)) === " ") || (symbolIdx === 2)) {
                                         const emojis = StatusQUtils.Emoji.emojiJSON.emoji_json.filter(function (emoji) {
-                                            if (emoji.aliases_ascii.includes(getText((cursorPosition - 2), cursorPosition)) ||
-                                                    emoji.aliases_ascii.includes(getText((cursorPosition - 3), cursorPosition))) {
-                                                var has2Chars = emoji.aliases_ascii.includes(getText((cursorPosition - 2), cursorPosition));
-                                                replaceWithEmoji("", getText(cursorPosition - (has2Chars ? 2 : 3), cursorPosition), emoji.unicode);
+                                            if (emoji.aliases_ascii.includes(getText((symbolIdx-1), (symbolIdx+1))) ||
+                                                emoji.aliases_ascii.includes(getText((symbolIdx-2), (symbolIdx+1)))) {
+                                                var has2Chars = emoji.aliases_ascii.includes(getText((symbolIdx-4), (symbolIdx-2)));
+                                                replaceWithEmoji("", getText((symbolIdx - (has2Chars ? 3 : 2)), symbolIdx), emoji.unicode);
```

Comment:
I still can't get the :beers: emoji; if I start typing `:b`, this gets immediately replaced with an unrelated emoji:

![image](https://user-images.githubusercontent.com/5377645/219096662-9507b1f4-f1c5-462a-bf50-2d3eb55400a8.png)

Reply:
I think it's an older issue. I can't send beers on 0.10

Reply:
It's hard to maintain both emoji shortName detection and ascii aliases. It's inevitable to have collisions while typing. Might be a design question on how it should work while typing.
I see Discord is using only shortName detection in the chat input, but the discord chat history will try to detect both ascii alias and short name. Sounds reasonable to me.

Reply:
I think both are doable; we'd just have to do the inline replacement when having typed `:b ` (ie. only after a space); I also noticed that Discord starts presenting the popup after having typed at least 2 characters from the emoji alias (ie. only after `:be`), not on the first char after colon (`:`)

Reply:
> > LGTM! Other than the `minimumHeight` update suggested by @caybro Might be worth creating another task to clarify the emoji detection while typing with design team. The Discord approach sounds better to me:
> > 
> > 1. Detect emoji short name in chat input
> > 2. Detect both short name and ascii alias in chat history
> > 
> > WDYT?
> 
> I'll repeat it here too: I think both are doable; we'd just have to do the inline replacement when having typed :b (ie. only after a space); I also noticed that Discord starts presenting the popup after having typed at least 2 characters from the emoji alias (ie. only after :be), not on the first char after colon (:)

Good point 👍. Sounds like a good rule